src/chart/heatmap/HeatmapView.ts

```diff
@@ -22,7 +22,7 @@ import { toggleHoverEmphasis } from '../../util/states';
 import HeatmapLayer from './HeatmapLayer';
 import * as zrUtil from 'zrender/src/core/util';
 import ChartView from '../../view/Chart';
-import HeatmapSeriesModel, { HeatmapDataItemOption } from './HeatmapSeries';
+import HeatmapSeriesModel, { BorderRadius, HeatmapDataItemOption } from './HeatmapSeries';
```

Comment:
The imported type `BorderRadius` is unused and can be removed here.

Reply:
ok